src/main/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolver.scala

```diff
@@ -56,13 +57,16 @@ private[marathon] class InstanceUpdateOpResolver(clock: Clock) extends StrictLog
         updateExistingInstance(maybeInstance, op.instanceId)(updater.reservationTimeout(_, clock.now()))
 
       case op: GoalChange =>
```

Comment:
Nit: What do you think about following the naming scheme and using the imperative `ChangeGoal` here?

Reply:
➕ 1

Reply:
I will do that on master... This was not introduced in this PR

Reply:
https://github.com/mesosphere/marathon/pull/6667